src/server/routes/collection.js

```diff
@@ -80,9 +140,23 @@ router.get('/:collectionId/edit', auth.isAuthenticated, auth.isCollectionOwner,
 
 router.post('/:collectionId/edit/handler', auth.isAuthenticatedForHandler, auth.isCollectionOwner, collectionCreateOrEditHandler);
 
-router.get('/:collectionId', (req, res) => {
+router.post('/:collectionId/remove', auth.isAuthenticated, auth.isCollectionOwnerOrCollaborator, async (req, res) => {
+	const {bbids} = req.body;
```

Comment:
We probably want to send a detailed error if there are no bbids (missing in body or empty array)